### ISSUE TITLE
liste géolocalisée d'enseignes de la grande distrib

### ISSUE THREAD

**Issue by c-o** [dataforgoodfr/croixrouge#9]
Nous aurons besoin de la liste géolocalisée des ensiegnes de la grande distrib en partenariat avec la croix rouge afin de les croiser avec la localisation des centres actuels ou ceux que nous recommanderions d'ouvrir.
Les enseignes principales actuelles sont:
carrefour, intermarche, super u, lidl, aldi, auchan, monoprix, atac, franprix, casino, leclerc

**Comment by martindaniel4** [comment]
@c-o @rquilliet @samronsin @RomainWarlop : une idée là dessus ? 

Ca vaudrait vraiment le coup de piger le maillage grande distrib / centre CRF

**Comment by RomainWarlop** [comment]
On peut crawler leur site ? Genre <a href="http://www.carrefour.fr/nos-magasins/liste-carrefour">cette liste</a> de Carrefour ?

**Comment by martindaniel4** [comment]
@RomainWarlop yes je pensais à ça, mais doit y avoir une liste structurée quelque part. Genre - http://www.c-radar.com/2014/09/drole-repartition-geographique-grande-distribution-en-france/

**Comment by RomainWarlop** [comment]
@martindaniel4 Pour avoir plus d'infos que ville + code postale ?

**Comment by martindaniel4** [comment]
@RomainWarlop bah ouai, ça serait bien d'avoir la localisation GPS exacte de chaque enseigne / magasin.

**Comment by RomainWarlop** [comment]
@martindaniel4  ok, on peut récupérer ça en crawlant aussi. Je fais un test dans l'aprem sur carrefour et tu me dis si ça te va ?

**Comment by martindaniel4** [comment]
@RomainWarlop :+1:

**Comment by martindaniel4** [comment]
@RomainWarlop tu peux référencer une pull request comme ça - https://github.com/dataforgoodfr/croixrouge/pull/13

Ca permet de dire : "Hé les gars, j'ai scrappé Carrefour, checkez ça https://github.com/dataforgoodfr/croixrouge/pull/13"

:beer:

**Comment by martindaniel4** [comment]
@RomainWarlop ouai mais du coup : 
- Y a pas tous les carrefours (230 parait pas énorme)
- Faudrait la latitude / longitude

**Comment by RomainWarlop** [comment]
En fait faut rajouter les city, express, etc ... mais bon au final c'est assez relou. J'vais chercher si y'a pas une meilleur liste quelque part

**Comment by martindaniel4** [comment]
ouai, faudrait surtout leur demander je pense. 

Sinon y a sûrement un truc déjà structuré quelque part

**Comment by martindaniel4** [comment]
@RomainWarlop @c-o @samronsin @rquilliet  - vous avez vu ça ? http://adresse.data.gouv.fr/

Pas mal pour nous !

**Comment by samronsin** [comment]
@martindaniel4 cool!
J'ai pensé à ça (pas encore trop regardé) pour récupérer de la data centres commerciaux: https://www.yelp.com/developers/documentation/v2/overview

**Comment by martindaniel4** [comment]
@samronsin yes bonne idée !

**Comment by deromemont** [comment]
Bonjour, je suis très intéressé par votre projet

Est ce que la recuperation depuis la DB de open street map peut être faite sinon ?
Je ne connais pas l'exhaustivité de open street map sur ce point

**Comment by d4gvince** [comment]
@deromemont : les grands esprits se rencontrent ;-)
Le jeu de données issu d'OSM est ici : 
https://github.com/d4gvince/geogrocery/tree/master/france

**Comment by deromemont** [comment]
@d4gvince Merci, quelle réactivité ;)

**Comment by ssoulier** [comment]
Ce site recense un grand nombre de supermarché : http://supermarches.grandes-enseignes.com/
On est une petite équipe chez Axa avoir trouvé un moyen assez rapide et simple de le scrapper grâce notamment au site : https://import.io/.
Le scrapping est terminé, il faut maintenant enlever les doublons potentiels et geocoder. La base est sur BQ à l'adresse : https://console.developers.google.com/m/cloudstorage/b/stojou/o/enseignes_france.csv?authuser=0

**Comment by martindaniel4** [comment]
:+1:  :+1: 

Mortel. Une fois qu'on aura avancé sur la partie prédiction de la demande, on pourra lier cela aux magasins les plus proches. 

@vjuju toujours chaud pour présenter ça le 30.06 ?

**Comment by vjuju** [comment]
Carrément. Ca ne sera vraiment pas très long. Je pense qu'un quart d'heure suffit à présenter ce que l'on a fait :). Je vais réfléchir aux usages et next steps possibles de cette base pour discussion.

**Comment by martindaniel4** [comment]
@vjuju ce qui serait top, ce serait de lier ça au travail de @RomainWarlop et @lefko2000 sur la prédiction de la demande - https://github.com/dataforgoodfr/croixrouge/issues/37#issuecomment-111719340

**Comment by vjuju** [comment]
@ssoulier tu peux partager ton code R sioupli (en enlevant ma clé import.io présente dans la requête à l'API si possible) ? Merci beaucoup !!

**Comment by vjuju** [comment]
@martindaniel4 Je regarde ça.

**Comment by ssoulier** [comment]
@vjuju J'ai poussé le code et retiré la userKey de l'url de l'API.

**Comment by ssoulier** [comment]
J'ai fait un peu de cleaning de la base des enseignes. On a maintenant 14642 supermarchés uniques répertoriés. J'ai aussi mis la base sur le repo et à jour sur google storage

**Comment by martindaniel4** [comment]
:+1:
